### ISSUE TITLE
Model v6：三分类 settlement 输出层与训练

### ISSUE THREAD

**Issue by phead198708** [phead198708/BiGan#93]
在改造后的 v6 settlement 语料基础上，新建一版三分类 settlement 模型（v6），输出独立的 `p_up / p_down / p_neutral`，消除当前 `1 - p_up` 语义问题。

## 工作内容

1. 输出层与任务定义
- 将现有二分类任务改为三分类：UP / DOWN / NEUTRAL。
- 模型输出 logits：
  - z_up, z_down, z_neutral
- 概率：
  - [p_up, p_down, p_neutral] = softmax([z_up, z_down, z_neutral])
- settlement head 专注最终方向，不掺入 volatility 含义。

2. 训练 pipeline 改造
- 接入 v6 settlement 训练视图，使用 label_settlement_3way 作为监督信号。
- 损失函数改为多分类交叉熵（或 XGBoost 多分类配置）。
- 特征工程尽量保持与当前版本兼容，便于对比 v5 与 v6 的收益来自标签/输出而非大规模 feature drift。

3. 概率校准与评估
- 分别评估/校准：
  - p_up, p_down；p_neutral 主要用于 no-trade 决策。
- 核心指标：
  - accuracy / log loss / ROC-AUC / PR-AUC / Brier / ECE
  - 高置信度区域表现（例如 p_neutral 较低子集）
  - 与当前 v5 在相同切片和执行规则下的收益/回撤对比（如有 backtest）。

4. 接口与执行层对接
- 新 inference 返回结构示例：
  ```json
  {
    "p_up": ...,
    "p_down": ...,
    "p_neutral": ...,
    "metadata": { "model_version": "v6" }
  }
  ```
- 建议执行规则示例：
  - 仅当 p_up 最大且 p_up > threshold_up 且 p_neutral < neutral_cap 时做多
  - 对 DOWN 同理
- 明确禁止执行层再用 1 - p_up 作为 down 置信度。

## 验收标准

- v6 在离线评估中核心指标优于当前版本（或至少不退步）。
- 有 v5 vs v6 对比报告，覆盖多个代表性 family/horizon。
- 提供可调用的 inference 接口，返回 p_up/p_down/p_neutral。
- 有一份面向执行层的简要文档，说明：
  - 新输出含义
  - 推荐阈值/使用方式
  - 与现有接口的不兼容点。

**Comment by phead198708** [comment]
## Review / suggestions

Output-layer plan is sound. Reinforcements grounded in the v5 experience:

**1. The metric of record must be cost-adjusted / account-cashflow PnL, not AUC/Brier.** This is the central v5 lesson: v5's offline AUC (15M ~0.69) and calibration looked fine, but both real capped runs were net negative (#87: account PnL -0.985, repeatable account-vs-theoretical gap ≈ -0.072/trade). So "核心指标优于当前版本" should be **defined as cost-adjusted backtest / account-PnL under the #87 go/no-go basis**, and the v5-vs-v6 backtest should be **mandatory, not "如有 backtest"**. Offline AUC improvement is necessary but not sufficient.

**2. Multiclass calibration needs a real method.** Per-family isotonic on a single binary prob (v5's approach) does not directly extend to 3-way. Use temperature scaling or Dirichlet calibration (or per-class one-vs-rest isotonic with renormalization so `p_up+p_down+p_neutral=1`), and report **per-class ECE** plus reliability curves. Keep v5's family-aware structure (per-family calibrator + global fallback).

**3. Feature parity is the right call** — holding features ≈ v5 isolates the gain to labels/output rather than feature drift. Please log the exact feature-set diff (should be empty) in the comparison report.

**4. Tie `threshold_up` / `neutral_cap` to net-of-cost EV**, swept on validation, not picked by eye. Cross-reference the cost basis used in `profit_gating_policy_replay`.

**5. Enforce the `1 - p_up` ban in code, not just docs.** Add the new serving payload (`p_up/p_down/p_neutral`) to the prediction output contract and add a serving/contract test that fails if any consumer derives down-confidence from `1 - p_up`. List the interface incompatibility explicitly for the executor.

**Acceptance additions:** v5-vs-v6 cost-adjusted backtest required and per-family; per-class ECE reported; serving contract + regression test for the new 3-way payload.
